app_flutter/lib/status_grid.dart

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+import 'result_box.dart';
+
+class StatusGrid extends StatelessWidget {
```

Comment:
Good to add a basic smoke test. It lowers the barrier for new tests in the future.

Reply:
Added! Left a TODO to remove the smoke test when useful tests are added.